content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -50,6 +50,10 @@ folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
    and [shared drives](https://support.google.com/a/users/answer/9310351)\*
    (these two can only be referenced by ID).
 
+   > ⚠️ Files and folders should be shared to specific users (or groups) in
```

Comment:
Just `Folder should be ...` should be enough?

Reply:
Agree, updated.